doc/architecture/units.md

```diff
@@ -0,0 +1,11 @@
+# Measurement units of Permaplant
```

Comment:
Is already part of solution.

Reply:
I don't think I understand this critique, can you elaborate please?

Reply:
I mean the file doc/architecture/04solution.md of #425